scripts/golangci.yml

```diff
@@ -0,0 +1,170 @@
+---
```

Comment:
This need to be modified at https://github.com/ceph/ceph-csi/blob/devel/scripts/golangci.yml.in#L113

Reply:
@Madhu-1 didnt I modify it already? :)

Reply:
@Madhu-1 rebased this PR, can you please take a look at this?  this will have conflict coming in each other PR getting in. So the ping.

Reply:
`scripts/golangci.yml` should not be included, it gets re-generated when `scripts/golangci.yml.in` has been modified.